src/substrate/filters/enricher.ts

```diff
@@ -66,6 +67,25 @@ export async function Enrich(
         }
       }
 
+      /**
+       * Offences Events
+       */
+      case EventKind.Offence: {
+        const [ offenceKind, opaqueTimeSlot, applied ] = event.data as unknown as [ Kind, OpaqueTimeSlot, boolean ];
```

Comment:
This should be `bool`, the polkadot type, not `boolean`, the js native type. You can convert from the former to the latter with `applied.isTrue`.